assets/js/header.js

```diff
@@ -0,0 +1,86 @@
+let logoHeader = document.querySelector("#logo_header")
+let myHeader = document.querySelector("header")
+let burgerClosedIcon = document.querySelector("#closed_burger_icon")
+let myFooter = document.querySelector("footer")
+let burgerMenue = document.querySelector("#burger_menue")
+let burgerOpennedIcon = document.querySelector("#openned_burger_icon")
+let myMain = document.querySelector("main")
+
+/**
+ * Appel de la fonction onload() qui vient charger les écouteurs d'évenements
+ */
+window.onload = onload();
+
+/**
+ * Fonction appelée au chargement de la page avec toutes les fonctionnalités et écouteurs d'évenements
+ */
+function onload() {
+
+    /**
+     * / Variable mediaquerry pour le menu burger
+     */
+    let myMedia = window.matchMedia("(min-width: 1090px)")
+
+    logoHeader.addEventListener("click", () => {
+        window.location = "../index.php"
+    })
+
+    burgerClosedIcon.style.cursor = "pointer"
+    burgerOpennedIcon.style.cursor = "pointer"
+
+    burgerClosedIcon.addEventListener("click", () => {
+        openingBurger()
+    })
+
+    burgerOpennedIcon.addEventListener("click", () => {
+        closingBurger()
+    })
+
+    myMediaQuerry(myMedia)
+
+    myMedia.addEventListener('change', () => {
+        myMediaQuerry(myMedia)
+    });
+}
+
+/**
+ * Fonction qui permet de fermer le menu burger si il est ouvert et que la fenêtre s'agrandit 
+ */
+function myMediaQuerry(myMedia) {
+    if (myMedia.matches) {
+        burgerOpennedIcon.style.display = "none"
+        burgerMenue.style.display = "none"
+        myFooter.style.display = "flex"
+        myMain.style.display = "block"
+    } else {
+        burgerClosedIcon.style.display = "flex"
+    }
+}
+
+/**
+ * Fonction qui se déclenche à l'ouverture du menu burger
+ */
+function openingBurger() {
```

Comment:
Think about toggling css classes instead of adding inline css

Reply:
Fixed in the incoming push, I gave up the toggle in this case and chose to .add & .remove
I couldn't figure it out properly and already gave it 2 hours